turborepo-tests/integration/package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "turborepo-tests-integration",
+  "name": "turborepo-tests-integration-common",
```

Comment:
@arlyon I think you might have missed a pnpm install here? I don't see this changed reflected in the lockfile